core/src/main/java/org/elasticsearch/indices/breaker/CircuitBreakerStats.java

```diff
@@ -52,6 +48,24 @@ public CircuitBreakerStats(String name, long limit, long estimated, double overh
         this.overhead = overhead;
     }
 
+    public CircuitBreakerStats(StreamInput in) throws IOException {
+        // limit is the maximum from the old circuit breaker stats for backwards compatibility
```

Comment:
I'm guessing it made sense when read with some backwards compatibility code we've since removed? I don't think it applies any more so I'd remove it so no one else is confused by it.